src/libs/ajax/GoogleStorage.ts

```diff
@@ -48,6 +113,24 @@ export type GCSListObjectsResponse = {
 const encodeAnalysisName = name => encodeURIComponent(`notebooks/${name}`)
 
 export const GoogleStorage = (signal?: AbortSignal) => ({
+  checkBucketAccess: async (googleProject, bucket, accessLevel) => {
```

Comment:
return types on these methods?

Reply:
wait... not seeing much typing at all...

Reply:
I don't know the return types for these functions. The emphasis here is on reorganizing functions, not functional changes, so I don't think types necessarily need to be added here. Same for the Billing module.